warehouse/ingest-scripts/src/main/resources/bin/ingest/generate-accumulo-config-cache.sh

```diff
@@ -12,12 +12,10 @@ cd $THIS_DIR
 
 export ACCUMULO_OTHER_OPTS="${ACCUMULO_OTHER_OPTS} $HADOOP_INGEST_OPTS"
 
-ADDJARS="$( cd ../../lib && pwd )/*"
-
-CLASSPATH=$ADDJARS $WAREHOUSE_ACCUMULO_HOME/bin/accumulo \
+CLASSPATH=$CLASSPATH $WAREHOUSE_ACCUMULO_HOME/bin/accumulo \
```

Comment:
Why not just export CLASSPATH, instead of `CLASSPATH=$CLASSPATH`?

Reply:
No good reason. I can change it.

Reply:
Done.